app/views/rubygems/show.html.erb

```diff
@@ -57,85 +57,5 @@
-          <span class="gem__downloads"><%= number_with_delimiter(@latest_version.downloads_count) %></span>
-        </h2>
-      </div>
-    <% end %>
-
-    <% if @latest_version.indexed %>
-      <h2 class="gem__ruby-version__heading t-list__heading">
-        <%= t '.bundler_header' %>:
-        <div class="gem__code-wrap">
-          <input type="text" class="gem__code" id="gemfile_text" value="<%= @latest_version.to_bundler %>" readonly="readonly">
-          <span class="gem__code__icon" id="js-gem__code--gemfile" data-clipboard-target="#gemfile_text">=</span>
-          <span class="gem__code__tooltip--copy"><%= t('.copy_to_clipboard') %></span>
-          <span class="gem__code__tooltip--copied"><%= t('.copied') %></span>
-        </div>
-      </h2>
-      <h2 class="gem__ruby-version__heading t-list__heading">
-        <%= t '.install' %>:
-        <div class="gem__code-wrap">
-          <input type="text" class="gem__code" id="install_text" value="<%= @latest_version.to_install %>" readonly="readonly">
-          <span class="gem__code__icon" id="js-gem__code--install" data-clipboard-target="#install_text">=</span>
-        </div>
-      </h2>
-      <h2 class="gem__ruby-version__heading t-list__heading">
-        <%= pluralized_licenses_header @latest_version %>:
-        <span class="gem__ruby-version">
-          <p><%= formatted_licenses @latest_version.licenses %></p>
-        </span>
-      </h2>
-    <% end %>
-
-    <h2 class="gem__ruby-version__heading t-list__heading">
-      <%= t('.required_ruby_version') %>:
-      <i class="gem__ruby-version">
-        <% if @latest_version.required_ruby_version %>
-          <%= @latest_version.required_ruby_version %>
-        <% else %>
-          <%= t('none') %>
-        <% end %>
-      </i>
-    </h2>
-
-    <% if @latest_version.required_rubygems_version != '>= 0' %>
-      <h2 class="gem__ruby-version__heading t-list__heading">
-        <%= t('.required_rubygems_version') %>:
-        <i class="gem__ruby-version">
-            <%= @latest_version.required_rubygems_version %>
-        </i>
-      </h2>
-    <% end %>
-
-    <h3 class="t-list__heading"><%= t '.links.header' %>:</h3>
-    <div class="t-list__items">
-      <%= link_to t('edit'), edit_rubygem_path(@rubygem), :class => "gem__link t-list__item", :id => "edit" if @rubygem.owned_by?(current_user) %>
-
-      <% if @latest_version.indexed %>
-        <% if @rubygem.linkset.present? %>
-          <%- Linkset::LINKS.each do |link| %>
-            <%= link_to_page link, @rubygem.linkset.public_send(link) %>
-          <%- end %>
-        <% end %>
-
-        <%= download_link(@latest_version) %>
-      <% end %>
-
-      <%= documentation_link(@latest_version, @rubygem.linkset) %>
-      <%= badge_link(@rubygem) %>
-      <%= subscribe_link(@rubygem) if @latest_version.indexed %>
-      <%= unsubscribe_link(@rubygem) %>
-      <%= atom_link(@rubygem) %>
-      <%= report_abuse_link(@rubygem) %>
-    </div>
-  </div>
```

Comment:
oh! :sweat_smile: Sorry @spk. I think we can use it.

Also, sorry again for delay with this PR. I look forward to getting this merged once I am sure about impact it might have on our servers. Checking out your branch locally :wink: 

Thank you so much for writing this https://www.spkdev.net/2015/05/23/reverse-dependencies-with-rubygems.html :heart: